crates/core/src/db/messages/commit.rs

```diff
@@ -1,13 +1,9 @@
 use super::transaction::Transaction;
-use crate::hash::Hash;
 use std::sync::Arc;
 
 // aka "Block" from blockchain, aka RecordBatch, aka TxBatch
 #[derive(Debug)]
 pub struct Commit {
-    pub parent_commit_hash: Option<Hash>,
```

Comment:
I understood this as an integrity protection mechanism, which is vital for replication. Can you explain what alternative you're offering?

Reply:
I want to emphasize that this is a draft PR for me to point out that some code is currently unused to a specific person.